tests/e2e/test_rosacli_ingress.go

```diff
@@ -116,11 +116,99 @@ var _ = Describe("Edit default ingress",
 					Expect(textData).Should(ContainSubstring("Updated ingress '%s' on cluster '%s'", defaultID, clusterID))
 				}
 
-				By("Run command to edit an default ingress with --label-match")
+				By("Run command to edit a default ingress with --label-match")
 				output, err = ingressService.EditIngress(clusterID, defaultID,
 					"--label-match", "aaa=bbb,ccc=ddd")
 				Expect(err).To(HaveOccurred())
 				textData = rosaClient.Parser.TextData.Input(output).Parse().Tip()
 				Expect(textData).Should(ContainSubstring("ERR: Updating route selectors is not supported for Hosted Control Plane clusters"))
 			})
+
+		It("Change load balancer type [id:64767]",
+			labels.Critical,
+			labels.Day2,
+			func() {
+				if isHosted {
+					Skip("This case is for standard ROSA clusters only")
+				}
+
+				output, err := ingressService.ListIngress(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				ingressList, err := ingressService.ReflectIngressList(output)
+				Expect(err).ToNot(HaveOccurred())
+				var defaultID string
+				for _, ingress := range ingressList.Ingresses {
+					if ingress.Default == "yes" {
+						Expect(ingress.LBType).Should(ContainSubstring("classic"))
+						defaultID = ingress.ID
+					}
+				}
+
+				output, err = ingressService.EditIngress(clusterID, defaultID, "--lb-type", "nlb")
```

Comment:
There is a potential risk that `defaultID=""`. You need to add an `Expect` before executing the step.